whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/NrtmVersionInformationDao.java

```diff
@@ -0,0 +1,92 @@
+package net.ripe.db.nrtm4.persist;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.sql.Statement;
+import java.util.Optional;
+import java.util.UUID;
+
+
+@Repository
+public class NrtmVersionInformationDao {
+
```

Comment:
rename to NrtmVersionInfoDao?

Reply:
actually changed in later PR to NrtmVersionInfoRepository -- coz i think it's cleaner to reflect the `@Repository` annotation in the name and also because DAO is logically separate in that it can use a repository to do data manipulation but not necessarily directly coupled to the db. think of it like a data service, or a thin layer on top of the database.